tests/test_json_regex.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+from json import loads, dumps
+from apluslms_roman.utils.path_mapping import json_re, env_value_to_dict
+
+test_case_loadable = (
+    True,
+    False,
+    None,
+    123,
+    -123,
+    3.14,
+    -3.14,
+    {'foo': 'bar'},
+    [1, 2, 3],
+    "foo bar"
+)
+
+test_case_not_loadable = (
+    "/foobar.py",
+    "text",
+    "yes",
+    "0123123",
+)
+
+
+def try_loads_json(json_str):
```

Comment:
I would prefer if this function would be removed and the code would use `self.assertRaises` instead, if that would be possible.